From fcba0df068d07ee7a26ec9d891220233d7d17dfd Mon Sep 17 00:00:00 2001
From: Paul Holzinger <paul.holzinger@web.de>
Date: Wed, 6 Jan 2021 23:32:40 +0100
Subject: [PATCH] Fix build for mips architecture followup

Followup to commit (1ad796677e1c). The build on mips is still
failing because SIGWINCH was not defined in the signal pkg.

Signed-off-by: Paul Holzinger <paul.holzinger@web.de>
---
 pkg/signal/signal_linux_mipsx.go | 2 ++
 1 file changed, 2 insertions(+)

--- a/pkg/signal/signal_linux_mipsx.go
+++ b/pkg/signal/signal_linux_mipsx.go
@@ -19,6 +19,8 @@ import (
 const (
 	sigrtmin = 34
 	sigrtmax = 127
+
+	SIGWINCH = syscall.SIGWINCH
 )
 
 // signalMap is a map of Linux signals.
